axon-framework/events/event-versioning.md

```diff
@@ -13,6 +13,13 @@ Upcasters are classes that take one input event of revision `x` and output zero
 > **Note**
 >
 > Perhaps the greatest benefit of upcasting is that it allows you to do non-destructive refactoring. In other words, the complete event history remains intact.
+>
+> **Conversion Notice**
```

Comment:
Text is super! But I think it's cleaner if it is segregated from this other nice a bit more.
What about adding it to the end of section `## Event Upcasting`, so just right before `### Provided abstract Upcaster implementations`?

Reply:
Changed it... let me know if this is what you had in mind.

Reply:
Ah, so my comment had a typo, talking about "nice" instead of "note"...
What I mean to say is that I think it is better that the `**Conversion Notice**` is not attached to the `**Note**` starting on line 13.

Reply:
Got it! Let me know what you think.